src/ngraph/builder/split.hpp

```diff
@@ -50,5 +50,43 @@ namespace ngraph
         /// \return     The vector containing multiple nodes we split input node into.
         ///
         NodeVector split(const Output<Node>& value, size_t split_parts, int axis = 0);
+
+        namespace opset1
+        {
+            /// \brief      Split value on specified axis into multiple parts.
+            ///
+            /// \param[in]  value         The value to be split.
```

Comment:
Since we're not using `\param[out]` here, I think we should just use the `\param` convention.

Reply:
done, I changed `\param[out]` to `\param`